test/system/profile_company_change_test.rb

```diff
@@ -43,7 +43,7 @@ class ProfileCompanyChangeTest < ApplicationSystemTestCase
     assert_text "Profile company updated."
 
     within_content_body do
-      assert_text :all, "Company Test Co."
```

Comment:
I was scratching my head about these changes for a minute. Now I see that the `title` argument was being injected into the SVG to serve as `<title>` tag within it.

Is there an accessibility impact? Do user agents read the `<title>` in the SVG? If so, are we losing an accessibility aid? Should these be replaced with an alt text? Not sure how to do that with Bootstrap icons, a quick search seems to suggest to use `.visually-hidden` within the `<i>`: https://getbootstrap.com/docs/5.3/getting-started/accessibility/#visually-hidden-content